<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dip_project2_GUIClass</class>
 <widget class="QMainWindow" name="dip_project2_GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dip_project2_GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>file road</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>face</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>car</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>GO</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="resultlist">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>61</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>이미지 결과</string>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>감지 시간:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelPath">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>2</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>파일경로</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fileSize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>파일 수</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>309</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="plainText">
     <string>2020년 디지털영상처리설계
TEAM PROJECT2
김현
정연희</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>276</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>280</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>저장 경로</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="dip_project2_GUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>dip_project2_GUIClass</receiver>
   <slot>openButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>dip_project2_GUIClass</receiver>
   <slot>processButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>dip_project2_GUIClass</receiver>
   <slot>selectedImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resultlist</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>dip_project2_GUIClass</receiver>
   <slot>selectedImage2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openButton()</slot>
  <slot>processButton()</slot>
  <slot>selectedImage()</slot>
  <slot>selectedImage2()</slot>
 </slots>
</ui>
